ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java

```diff
@@ -65,6 +67,7 @@ public void contactInfoTest() throws Exception {
         patientDashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
         patientDashboardPage.clickOnShowContact();
         assertTrue(patientDashboardPage.getTelephoneNumber().contains("111111111"));
+        patientDashboardPage.waitForPage();
```

Comment:
Any reason for adding waiter here?

Reply:
The reason behind this waiter is that patientDashboardPage is taking long to be executed fully, i tested this on two instances , the running host will quit with that waiter, but on qa server it really takes long , thats why i added that waiter